docs/controls/Line.md

```diff
@@ -193,6 +194,17 @@ Each line can have a style object in [Values](#values) level.
 | --------------- | -------- | ------- | ------------------------------------------------------ |
 | strokeDashArray | string   | "0"     | Applies stroke-dasharray CSS property to the SVG line. |
 
+##### LegendOptions
+
+Each line cna have a legendOptions object in [Values](#values) level.
```

Comment:
Everything looks good to me. Just a minor change for `can`.

Reply:
I have updated [here|https://github.com/SharathAlur/carbon-graphs/commit/b9324c1e209febc4aaffbdbd235f2a6812b10f5d]